browse-scaler/src/__test__/resize.test.js

```diff
@@ -1,55 +1,71 @@
-const fs = require('fs');
-const sharp = require('sharp');
-const { resizeImage, notFound } = require('../resize');
+const fs = require("fs");
+const sharp = require("sharp");
+const { resizeImage, notFound } = require("../resize");
 
-describe('Image resizing', () => {
-  test('Images are resized as expected', async () => {
-    const image = fs.readFileSync('stars.jpg');
+describe("Image resizing", () => {
+  test("Images are resized as expected", async () => {
+    const image = fs.readFileSync("stars.jpg");
 
     const imageResponse = await resizeImage(image, 200, 200);
-    fs.writeFileSync('stars-resize.png', imageResponse, { encoding: 'base64' });
+    fs.writeFileSync(__dirname + "stars-resize.png", imageResponse, {
```

Comment:
I think this should be `__dirname + '/stars-resize.png'`, or better yet `path.join(__dirname, 'stras-resize.png')`